nautobot/ipam/tests/test_api.py

```diff
@@ -610,7 +602,7 @@ def test_ports_regression(self):
             "name": "http",
             "protocol": "tcp",
             "device": str(device.id),
-            "ports": ["80"],
+            "ports": [80],
```

Comment:
Concerned about this change given that the test case appears to be explicitly about checking that a str is okay??